src/pages/DirectMessage.tsx

```diff
@@ -29,8 +31,36 @@ import UserAvatar from "../components/Common/UserAvatar";
 import EmojiPickerIconButton from "../components/Common/EmojiPickerIconButton";
 import { EmojiClickData } from "emoji-picker-react";
 import TooltipTimestamp from "../components/Common/TooltipTimestamp";
+import formatTimestamp from "../utilities/formatTimestamp";
+import PageLoader from "./PageLoader";
```

Comment:
This is currently not used. Is there a reason we're using `CircularProgress` over our component? Is it the styling?

Reply:
It was the styling initially, but because I wrapped the implementation in a box with style `bioContainer`, the styling shouldn't affect it. I will revert to using `PageLoader` as implemented throughout the app

Reply:
Ok nice, maybe we should rename `PageLoader` to just `Loader` sometime lol